src/axios-har-tracker.ts

```diff
@@ -116,31 +137,23 @@ async function axiosTracker(call) {
     return obj ? results : [];
   }
 
-  function getCookies(fullCookie: string) {
+  private getCookies(fullCookie: string) {
     if (fullCookie) {
```

Comment:
fullCookie is a mandatory parameter so it will be always defined, no need for checking with if statement